src/views/view_game.py

```diff
@@ -73,9 +73,55 @@ def init_game_btns(self):
             pos=(1100, 610),
             width=80,
             height=80,
-            text="Quit",
+            text="(6) Quit",
             background=(180, 180, 180),
         )
+
```

Comment:
Delete this empty line (too much blank lines)

Reply:
done